csharp/src/Ice/INetworkProxy.cs

```diff
@@ -148,7 +148,7 @@ public EndPoint GetAddress()
 
         public string GetName() => "SOCKS";
 
-        public int GetProtocolSupport() => Network.EnableIPv4;
+        public int GetIPVersion() => Network.EnableIPv4;
```

Comment:
We should consider renaming EnableIPv4 constant, sounds a bit odd to use it as the return of GetIPVersion